server/build.sbt

```diff
@@ -145,12 +146,23 @@ lazy val root = (project in file("."))
   )
   .settings(excludeTailwindGeneration: _*)
 
+  //jacoco report setting
+jacocoReportSettings := JacocoReportSettings()
+  .withTitle("Jacoco")
+  .withFormats(JacocoReportFormats.CSV)
+//html for local views
+jacocoReportSettings := JacocoReportSettings()
```

Comment:
I'm pretty sure this is overriding the CSV version.  I think you can do `withFormats(JacocoReportFormats.CSV, JacocoReportFormats.HTML)`

Reply:
(see https://github.com/sbt/sbt-jacoco/blob/eb82c08f694e9e79c810906e8ad6420f8c4987a5/src/sbt-test/sbt-jacoco/report-settings/build.sbt)